### DISCUSSION TITLE
Welcome to The-Standard Discussions!

### DISCUSSION THREAD

**Discussion by hassanhabib** [hassanhabib/The-Standard#216] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AbuHuraira537** [comment]
Introduction ->
Name : Abu Huraira 
I am a Software Engineer. My plan is at first I will learn the The Standard of this community, and in future if I want to be able to contribute in it. Open source community projects are very large and hard to understand for newbies, Here I find some small Open source projects where I can start to contribute. My aim in future to fully understand the soul of these standard and contribute in it as well as in Open source community.

I want to join this community where I can contribute in it as I get benefit from it and I want give benefits to either in way of contributing in open source projects or in the other way.
